packages/ui5-info/README.md

```diff
@@ -0,0 +1,33 @@
+# @sap-ux/ui5-info
+
+Gets UI5 version info for available UI5 versions.
+
+## Installation
+Npm
+`npm install --save @sap-ux/ui5-info`
+
+Yarn
+`yarn add @sap-ux/ui5-info`
+
+Pnpm
+`pnpm add @sap-ux/ui5-info`
+
+
+## Usage
+
+```javascript
+import { getUI5Versions, type UI5VersionFilterOptions } from '@sap-ux/ui5-info';
+
+const filterOptions: UI5VersionFilterOptions = {
+    useCache: true,
+    removeDuplicateVersions: true,
+    groupUI5Versions: true
```

Comment:
These options do not exist on `UI5VersionFilterOptions` anymore

Reply:
Done thx